src/runtime_src/core/common/device.h

```diff
@@ -24,6 +24,7 @@
 #include "scope_guard.h"
 #include "uuid.h"
 #include "core/include/xrt.h"
+#include "core/common/drv/include/mem_group.h"
```

Comment:
Does mem_group.h need to be included in header file (device.h)?

Reply:
Ok. I will move it to device.cpp file.